test/plugins/sqlite.js

```diff
@@ -22,25 +22,28 @@ describe("SQLite Message Storage", function() {
 		}
 	});
 
+	after(function(done) {
+		store.close((err) => {
+			expect(err).to.be.null;
+			expect(fs.existsSync(expectedPath)).to.be.true;
```

Comment:
Expects in after? This is becoming increasingly bizarre lol. We're not really supposed to do that

Reply:
`after` can be a test just like any other, no? we're just guaranteeing the order here.

Reply:
I've noticed in the past that the reporter usually lose it when there is a test in a before/after hook.
Because tests in this file **need** to run in order anyway, and this effectively runs tests, I would move it in a `it` statement at the very bottom of the file.

Reply:
Does `expect(fs.existsSync(expectedPath)).to.be.true;` test that the file wasn't destroyed when exiting or something? Do we want/need to test the file is valid, and/or not empty, and/or load, etc.?

Reply:
It's testing that the file is created at the expected path.

Reply:
Ah, makes sense. Then in an `it` at the end should do, no?

Reply:
Done.